website/gatherDocs.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+const fs = require("fs-extra");
+const git = require("nodegit");
+const execSync = require('child_process').execSync;
+const REPO_URL = "https://github.com/wix/detox.git"
+
+// From https://gist.github.com/joerx/3296d972735adc5b4ec1
+function clearRequireCache() {
+  Object.keys(require.cache).forEach(function (key) {
+    delete require.cache[key];
+  });
+}
+
+function getMajorVersion(tag) {
```

Comment:
Please use 'semver' instead of parsing versions manually, this will break the pretty easily

Reply:
Good idea 👍

Reply:
I fixed that @rotemmiz 👍 Please review again!